tests/test_projectgezond.py

```diff
@@ -18,7 +18,7 @@ def test_title(self):
         self.assertEqual("Boeuf bourguignon", self.harvester_class.title())
 
     def test_category(self):
-        self.assertEqual("Diner\nKerstrecepten", self.harvester_class.category())
+        self.assertEqual(["Diner", "Kerstrecepten"], self.harvester_class.category())
```

Comment:
Ah, this helped to uncover a related problem: calls to `scraper.category()` should return a string type, not a `list` (as here).

We don't have type hints in place to make that clearer, but there is work-in-progress to add those.

Reply:
The fix looks good, thanks!